pkgs/applications/radio/tqsl/default.nix

```diff
@@ -9,7 +9,7 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-UGPMp1mAarHWuLbZu2wWpjgCdf8ZKj0Mwkqp32U5/8w=";
   };
 
-  nativeBuildInputs = [ cmake makeWrapper ];
+  nativeBuildInputs = [ cmake makeWrapper wrapGAppsHook ];
```

Comment:
Why was makeWrapper needed?

Reply:
Was not changed from original code. Did not want to change too much as still learning all of the nixifications.

As not being called I have updated the code to remove unused wrapper, and update db to lmdb.